adafruit_pcf8591/analog_out.py

```diff
@@ -26,12 +26,16 @@
 
 * Author(s): Bryan Siepert
 """
+try:
+    import typing  # pylint: disable=unused-import
+except ImportError:
+    pass
 
 
 class AnalogOut:
     """AnalogIn Mock Implementation for ADC Reads."""
 
-    def __init__(self, pcf, dac_pin=0):
+    def __init__(self, pcf: "PCF8591", dac_pin: int = 0) -> None:
```

Comment:
Would you mind fixing the docstring here as well?  Thanks!

Reply:
also made a change in doc string to A0 instead of P4, since that resolves to 0